SQL/0000-00-00-schema.sql

```diff
@@ -448,6 +448,7 @@ CREATE TABLE `files` (
   `ProcessProtocolID` int(11) unsigned,
   `Caveat` tinyint(1) default NULL,
   `TarchiveSource` int(11) default NULL,
+  `ScannerCandID` int(11) DEFAULT NULL,
```

Comment:
I think this should be a foreign key to the ID column, and not the CandID column of the mri_scanner table.